packages/react-native-sdk/src/providers/ActiveCallManager/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useAndroidKeepCallAliveEffect } from '../../hooks';
+import { useProcessPushCallEffect } from './useProcessPushCallEffect';
+
+/**
+ * A renderless component that does two things
+ * 1. keeps the call alive in background in Android
+ * 2. processes the incoming call push notification
+ * This component must be a child of StreamVideo
+ * @internal
+ */
+export const ActiveCallManager = (): React.ReactElement | null => {
```

Comment:
Why can't we call these hooks directly in the ActiveCall component? A component simply returning null and call hooks internally doesn't look good to me, may be convert it to a use `useActiveCall`.

Reply:
Can we reply before resolving @khushal87 has a question 👀

Reply:
Now its moved it to StreamCall, we had a discussion on slack about not keeping business logic in ActiveCall component to allow for customisation so did not move it there..

since the comment is valid but it was moved to StreamCall I pressed on resolve.. apologies for the confusion if any